public/src/public/template.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Reddit Archive</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+  <link href="https://www.redditstatic.com/mweb2x/favicon/64x64.png" rel="icon shortcut" sizes="64x64">
+</head>
+<body>
+<style>
+  body {
+    background: #d9dfe5;
+  }
+  .spinner {
+    position: absolute;
+    top: 42%;
+    left: 45%;
+    width: 70px;
+    z-index: 10;
+  }
+</style>
+<div id="app">
+  <img class="spinner" src="http://127.0.0.1:8888/spinner.gif" alt="spinner">
```

Comment:
Replace it with css-animated div? =)

Reply:
)

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/6b71b140a52676a0fac5143c0bc4135d52f7b717